nbs/examples/Large_Datasets_Tutorial.ipynb

```diff
@@ -0,0 +1,691 @@
+{
```

Comment:
<p>Can you please add a markdown cell below this one with the following spark example?</p><p> </p><blockquote>You can also create this directory structure from a spark dataframe with the following:</blockquote>

<pre class="ql-syntax" spellcheck="false">spark.conf.set("spark.sql.parquet.outputTimestampType", "TIMESTAMP_MICROS")
(
&nbsp; spark_df
&nbsp; .repartition(id_col)
&nbsp; .sortWithinPartitions(id_col, time_col)
&nbsp; .write
&nbsp; .partitionBy(id_col)
&nbsp; .parquet(out_dir)
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>